RecoLocalMuon/GEMRecHit/interface/ME0RecHitBaseAlgo.h

```diff
@@ -48,6 +48,7 @@ class ME0RecHitBaseAlgo {
                              LocalPoint& Point,
                              LocalError& error) const = 0;
 
+  bool recOnlyMuons;
```

Comment:
I think this one should have been removed as well.
(an exposed public data member by itself is not good)